### ISSUE TITLE
Payments row is created but no donationId

### ISSUE THREAD

**Issue by tiagonmas** [banco-alimentar/alimentestaideia.pt#737]
There are payment rows that are "orphan" and do not have a related donation.
Last row with null donationid is from 8th of Dec 2022.

```
select * from payments where DonationId is null
select count(1) from payments where DonationId is null
```

**Comment by guerrerotook** [comment]
I created a tool that consolidates this information. Out of the 72 rows, 40 are from donation completed but are not linked, so I updated the database with this tool. 

The rest are donations coming from a Subscription that the user didn't finish, so we don't have any way to link that to a Donation. 

There are also some other Payments with the same, TransactionKey and EasyPayPaymentId because they were multiple try from the user, but one of the is succeed and the rest and not completed. We can deleted those if you want.
